packages/usb/usb.1.3.1/opam

```diff
@@ -0,0 +1,50 @@
+opam-version: "2.0"
+name: "usb"
+version: "1.3.1"
+authors: "Jeremie Dimino <jeremie@dimino.org>"
+maintainer: "letoh"
+license: "BSD-3-Clause"
+homepage: "https://github.com/letoh/ocaml-usb"
+bug-reports: "https://github.com/letoh/ocaml-usb/issues"
+dev-repo: "git+https://github.com/letoh/ocaml-usb.git"
+
+depends: [
+  "ocaml" {>= "4.01.0"}
+  "ocamlfind"
```

Comment:
Do you use this at runtime or is it a build dependency?

Reply:
Oasis needs it, so maybe I should leave it to oasis? or just add the build filter?